app/controllers/users_controller.rb

```diff
@@ -1,9 +1,18 @@
+require 'set'
+
 class UsersController < ApplicationController
   include SocialAccountsHelper
-  before_action :authorize
-  helper_method :generate_link, :supported_social_networks
-
-  def show; end
+  include UsersHelper
+  before_action :authorize, except: %i[show index search]
```

Comment:
Why do you change the authorization settings for this controller? I couldn't find it specified in the linked ticket and I think @LinqLover did really good work when introducing proper authentication.

Reply:
Not sure why I did it. I removed it.